app/utils.py

```diff
@@ -0,0 +1,11 @@
+import re
+from flask_mail import Message
+
+def validate_email_format(email):
+    pattern = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+    return re.match(pattern, email) is not None
+
+def send_verification_email(email, mail):
+    msg = Message('Verification Email', sender='ahossain3324@gmail.com', recipients=[email])
```

Comment:
Lets say, for now we are sending a verification email, but we may also need to send a verification code to the phone. Sending notifications or verification codes can be a separate task. So, we can use a separate class, something like `VerficationService` to do this job, and then we can call this class method from where necessary.  So, better we can keep it separate. Also, we will put those functions in `Helper` or `Utils` class that are frequently used throughout our code base.